src/QuartzNET-DynamoDB.Tests/Unit/DynamoTriggerStateTests.cs

```diff
@@ -68,6 +68,28 @@ public void ComparisonTrueIfRightNull()
             Assert.True(sut != null);
         }
 
+		/// <summary>
+		/// Tests that when checking a null refenence of type DynamoTriggerState with null, true is returned. 
+		/// I did not expect this to invoke the overridden operator in the DynamoTriggerState class but it does!
+		/// </summary>
+		[Fact]
+		[Trait("Category", "Unit")]
+		public void TestReferenceBothNull()
+		{
+			DynamoTriggerState thing = null;
```

Comment:
@ddhi004 added this test because it was failing until I made this fix: https://github.com/lukeryannetnz/quartznet-dynamodb/blob/fb5a8aa2327270aefa2cf97094c5119fb5d4afe7/src/QuartzNET-DynamoDB/DataModel/DynamoTriggerState.cs#L115-L118

Discovered due to this test (which I had previously deleted before merging from master): https://github.com/lukeryannetnz/quartznet-dynamodb/blob/fb5a8aa2327270aefa2cf97094c5119fb5d4afe7/src/QuartzNET-DynamoDB.Tests/Unit/DynamoTriggerTests.cs#L32